libs/zsdl/src/sdl2.zig

```diff
@@ -103,6 +103,24 @@ pub fn makeError() error{SdlError} {
 // Querying SDL Version
 //
 //--------------------------------------------------------------------------------------------------
+pub const Version = extern struct {
+    major: u8,
+    minor: u8,
+    patch: u8,
+};
+
+/// Compiled SDL version
+pub const VERSION = Version{
+    .major = 2,
+    .minor = 24,
+    .patch = 1,
+};
+
+/// Returns the linked SDL version
+pub fn getVersion(version: *Version) void {
```

Comment:
We could be more consistent with other bindings here if we don't pass a ptr to Version and instead construct and return from the fn. Generally we only stick to the C paradigm when it's valid to pass null, which is not the case here.
The signature would be:
`pub fn getVersion() Version`
What do you think?

Reply:
I like it :+1: